### ISSUE TITLE
Arrange initial SNOINC discussions w/key internal stakeholders

### ISSUE THREAD

**Issue by GogsByrn** [NHSDigital/FHIR-Pathology#79]

**Comment by GogsByrn** [comment]
From: TURNBULL, Pete (NHS DIGITAL) <pete.turnbull@nhs.net> 
Sent: 10 October 2022 10:25
To: MOUNTFORD, Richard (NHS DIGITAL) <richard.mountford@nhs.net>; ABBOTT, Stuart (NHS DIGITAL) <stuartabbott@nhs.net>; DL-ClinicalTerminology <nhsdigital.DL-ClinicalTerminology@nhs.net>
Cc: DONOHER, Lee (NHS DIGITAL) <lee.donoher3@nhs.net>; REES, Karen (NHS DIGITAL) <karen.rees15@nhs.net>; CHEETHAM, Ed (NHS DIGITAL) <ed.cheetham@nhs.net>; ROGERS, Jeremy (NHS DIGITAL) <jeremy.rogers@nhs.net>; ELIAS, Alex (NHS DIGITAL) <alex.elias@nhs.net>; PERRY, Andrew (NHS DIGITAL) <andrew.perry8@nhs.net>
Subject: RE: Tweet by Charles Gutteridge

Hi Richard,

Yep, nothing should stop us completing the current plan, the PBCL and the UTL still needs modelling no matter what we do next. However, what we do with the UTL/PBCL when it’s good enough to show to the world, and what we do around content we are yet to author might well needs a re-think, and anyone involved in or around this work can feed into that. 

Rather than more emails we’ll move this discussion into MS Teams. If anyone wants to share their wisdom with me 121 before Karim gets back, please fling something in my diary. Otherwise, you’ll get an invite to the Path Team led discussion when it starts.

**Comment by GogsByrn** [comment]
From: PERRY, Andrew (NHS DIGITAL) <andrew.perry8@nhs.net> 
Sent: 10 October 2022 11:57
To: TURNBULL, Pete (NHS DIGITAL) <pete.turnbull@nhs.net>; MOUNTFORD, Richard (NHS DIGITAL) <richard.mountford@nhs.net>; ABBOTT, Stuart (NHS DIGITAL) <stuartabbott@nhs.net>; DL-ClinicalTerminology <nhsdigital.DL-ClinicalTerminology@nhs.net>
Cc: DONOHER, Lee (NHS DIGITAL) <lee.donoher3@nhs.net>; REES, Karen (NHS DIGITAL) <karen.rees15@nhs.net>; CHEETHAM, Ed (NHS DIGITAL) <ed.cheetham@nhs.net>; ROGERS, Jeremy (NHS DIGITAL) <jeremy.rogers@nhs.net>; ELIAS, Alex (NHS DIGITAL) <alex.elias@nhs.net>
Subject: RE: Tweet by Charles Gutteridge

The details/politics of the LOINC SNOMED agreement are a bit difficult to decipher but things may be clearer, as Stuart says, after the LOINC meeting at the end of the month. 

During the SNOMED business, at the Pathology & Laboratory Medicine Clinical Reference Group there was a proposal from Jim Campbell to create laboratory content for pathology in community area using the LOINC-SNOMED post-coordinated expressions developed as part of the LOINC SNOMED agreement. There were representatives from LOINC in the meeting who were noticeably silent on the IP issues that have previously been an issue.

Then later, at the Expo there was the joint presentation by LOINC and SNOMED International referred to in Charles’ tweet. In this they announced that they would be creating a LOINC extension that would be hosted in the Community content area that would contain a SNOMED CT and LOINC identifier for every concept. The modelling of the LOINC content would be based on the SNOMED expressions. They expect this to be developed during 2023. From the LOINC perspective, it would seem that a key part of this, is that the content is developed and owned by LOINC which means that their US government funding is continued. I have attached the PDF of the presentation which can be viewed by registering via this zoom link. All-In-One Virtual Event Platform | Zoom Events

**Comment by GogsByrn** [comment]
From: ROGERS, Jeremy (NHS DIGITAL) <jeremy.rogers@nhs.net> 
Sent: 10 October 2022 12:16
To: PERRY, Andrew (NHS DIGITAL) <andrew.perry8@nhs.net>; TURNBULL, Pete (NHS DIGITAL) <pete.turnbull@nhs.net>; MOUNTFORD, Richard (NHS DIGITAL) <richard.mountford@nhs.net>; ABBOTT, Stuart (NHS DIGITAL) <stuartabbott@nhs.net>; DL-ClinicalTerminology <nhsdigital.DL-ClinicalTerminology@nhs.net>
Cc: DONOHER, Lee (NHS DIGITAL) <lee.donoher3@nhs.net>; REES, Karen (NHS DIGITAL) <karen.rees15@nhs.net>; CHEETHAM, Ed (NHS DIGITAL) <ed.cheetham@nhs.net>; ELIAS, Alex (NHS DIGITAL) <alex.elias@nhs.net>
Subject: RE: Tweet by Charles Gutteridge

Many thanks Andrew.

The plan does indeed appear to revolve chiefly around the creation of a new fully-fledged extension of SNOMED in which substantially all of LOINC is to become (somehow!) represented in parallel as a bona fide RF2 encoding of same. As a result most, perhaps all, LOINC identifiers will for the first time have an exhaustive and authoritative 1:1 bidirectional map to and from a legitimate precoordinated SNOMED component identifier. But since the ownership and meaning of all those precoordinated identifiers will likely remain with Regenstrief, this doesn’t actually change much: its ultimately just an alternate id and encoding for the exact same content and intellectual property, but now also in a SNOMED-stylie. 

The only genuinely new content that will likely result from this modest shift will be some kind of pre-computed IS-A relationships holding between these new LOINC-as-SNOMED identifiers and both each other and some of the existing SNOMED identifiers for concepts in the International Core, as a result of automatic classification of the LOINC OWL axioms. This is all therefore pretty much as Nebraska have already done, and Jim Campbell’s question from the floor at the formal session was therefore revealingly only neutrally welcoming. The agreement may have the effect of making much of his work to date redundant – but if it also now legitimises the bit of Jim’s work in which he creates entirely new lab test concepts and integrates them within LOINC, then I imagine he will be pleased overall.

Presumably however, these new precoordinated SNOMED conceptids as alternate handles for existing LOINCs will all be created within a new Regenstrief namespace, and will also be published initially only within this new Regenstrief-owned LOINC extension of SNOMED. It will presumably become OK to use either the original LOINC ID or its SNOMED Component counterpart interchangeably, both in local storage and or interop messages. This is currently verboten, and does represent a significant relaxation of Regenstrief licensing paranoia.

The main winners from an arrangement like this would be LOINC themselves, and the community of SNOMED implementors.

From a technical point of view, LOINC very badly need to replatform themselves onto something with both a well-designed and stable release format and a highly principled system of classification: LOINC’s existing table structure has long been a catastrophically denormalised mess; the core LOINC table comprises around 46 columns - it changes from release to release - and this is surrounded by an equally chaotic penumbra of accessory tables. Mind blowingly bad. They also need to find a way to move LOINC from being an informal frame-based system (broadly, a la CTV3) with only a very flat or often entirely absent hierarchy, to a formally specified and classified ontology (a la SNOMED). LOINC singularly lacks any kind of useable deep multiaxial reporting/analytics taxonomy, and they’ve only very recently thought to stare into the abyss of this.

Meanwhile, and also from a technical point of view, the world of interops has found that - precisely because SNOMED’s release mechanism is quite well-engineered - it would hugely simplify their implementation task if everything presented itself as part of SNOMED. Having to e.g. migrate anything to do with lab testing backwards and forwards to LOINC in order to satisfy the licensing rules would be a PITA even if LOINC’s own release files were simpler and more stable than they empirically are. Finding a way to re-represent LOINC instead entirely as “just another bit of the SNOMED universe” is therefore very attractive to the implementation community. Its not unlike Blighty’s own “Native XML dm+d” vs. “SNOMEDised dm+d” problem that is inexorably trending toward ditching the XML variant.

(Of course, one of the problems with the “Everything as SNOMED (One Ring to Rule Them All)” mantra is that whilst you could publish e.g. much of the content behind either Google Maps or The Radio Times entirely as a set of SNOMED codes, terms, relationships and subsets, this wouldn’t make either part of the SNOMED ontology: you couldn’t apply the same reasoning logic to them. Only pain will follow if you make the mistake of having un-reasonable content freely mixed in with reason-able content at any point when you choose to apply your reasoner. This is a technical challenge for the SNOMED community: whilst we should welcome the simplification that a common encoding and publishing paradigm would definitely bring, implementors need to understand that not everything published that way will be pieces of The Same Thing.)

Many unanswered questions remain despite the apparent narrowing of any fundamentally purely technical gap between LOINC and SNOMED. In particular, what does all this mean for the existing semantic overlap between the content contained within both of these two long-competing terminology products? For example, what happens to the substantial numbers of LOINCs that manifestly aren’t “observables” but that seem instead to directly correspond to other SNOMED chapters? 

There is, for example, a huge swathe of around 6000 LOINCs that appear to overlap closely with our own NICIP procedures:
… another block that looks a lot like Record Artifacts:
…and more than 15,000 LOINC Answers (in a GIANT flat list) amongst which are many that look an awful lot like SNOMED findings, disorders, body structures, environments, social context, qualifiers, morphologies etc.
And then there’s the more batshit-crazy stuff in LOINC:
Ultimately, if I’m correct about what this latest twist in the saga really turns out to be, it won’t actually change our own underlying intellectual property or governance issues at all: we probably still can’t borrow, recycle or otherwise compare notes with LOINC intellectual property drawn either from this New Work or from Nebraska’s extension in order to then build our own free-standing and unlinked competitor for the exact same content. In the same way -  and exactly as before -  we would also remain reluctant to recommend that the UK simply give up its current UTL ambitions in favour of adopting LOINC instead, because:
1)	We still don’t trust or respect how LOINC content development is governed: it’s still an unprincipled and not very transparent user-is-always-right free-for-all
2)	We would still have to work out how to map PBCL to it, but now whilst having no control of the map target
3)	We’d probably still have to author our own UK-Pathology-Mafia-Approved descriptions to layer over any Regenstrief content
4)	Full-fat LOINC has huge areas of poorly understood content overlap with SNOMED proper

There might eventually be a small PBCL-shaped fragment of LOINC that we’d be interested in, but it sounds as though this will only be made available as part of the much larger mudpie of all the other LOINC content (see above), much of which we very definitely want to avoid. I guess it is possible that in time they might move to componentise LOINC as a series of smaller, optional, separable and plug-n-play content units, one of which might become very UTL-shaped – and this would be a very sensible way to try and manage and track any incremental migration of LOINC from the mudpie it is now to what it could become. Indeed, Jim Campbell’s proposal to the Member Forum on the back of the announcement could be seen as a request to accelerate precisely this bit of work, at least in respect of the PBCL-shaped part. But I would suspect that’s probably still too far off for our immediate tactical needs, and so for now it is likely to be very difficult to extract just the UTL bits we want from any more comprehensively SNOMEDised LOINC.

The most likely scenario therefore remains that we still need to press ahead with building UTL pretty much as conceived, but it is now even more likely than it already was that our 5-10 year strategy should assume that things will eventually ground out in convergence with and then replacement by some well-defined part of Regenstrief-managed content. Viewed in that context, completing the currently scoped work on PBCL-to-UTL more or less as planned is actually entirely consistent with that plan for future convergence. Where things might become in increasing danger of being derailed would in fact be if UTL proved to be such a roaring success that there was UK pressure to expand its scope and remit. The resulting convergence task could then easily become increasingly difficult to carry off because of the scale and diversity of content needing to be converged and (likely) the scarcity of authoring resource available to apply to it.

**Comment by GogsByrn** [comment]
From: TURNBULL, Pete (NHS DIGITAL) <pete.turnbull@nhs.net> 
Sent: 10 October 2022 12:28
To: ROGERS, Jeremy (NHS DIGITAL) <jeremy.rogers@nhs.net>; PERRY, Andrew (NHS DIGITAL) <andrew.perry8@nhs.net>; MOUNTFORD, Richard (NHS DIGITAL) <richard.mountford@nhs.net>; ABBOTT, Stuart (NHS DIGITAL) <stuartabbott@nhs.net>; DL-ClinicalTerminology <nhsdigital.DL-ClinicalTerminology@nhs.net>
Cc: DONOHER, Lee (NHS DIGITAL) <lee.donoher3@nhs.net>; REES, Karen (NHS DIGITAL) <karen.rees15@nhs.net>; CHEETHAM, Ed (NHS DIGITAL) <ed.cheetham@nhs.net>; ELIAS, Alex (NHS DIGITAL) <alex.elias@nhs.net>
Subject: RE: Tweet by Charles Gutteridge

Hi,

Appreciate the whole LOINC thing is perhaps a distraction. What might be more interesting is the proposal from Jim…… which might be an opportunity to donate the “100% LOINC stain free” UTL to an international group of member organisations as a starting point for this work, and then use that group to massively accelerate our ability to author content as it would not just be us doing it. However, Karim had various conversations about this in Lisbon, and then has gone on leave (because hey, he was in Lisbon!) so worth waiting to see what was said before we decide anything probably.

**Comment by GogsByrn** [comment]
(karim's reply)

Hello,

I think Jim's slide speaks on behalf of the new agreement with Regenstrief, given that Stan Huff and Marjorie Rallins were in the room during the presentation and discussion. The scope is therefore a Regenstrief-owned SNOMED CT extension with content as prioritised by member counties, and doesn’t seem to be semantically / logically replicating the entirety of LOINC along with all of its character flaws.

Agree, the details are fuzzy so we need to convene and figure out how we collaborate with the effort and stay visible. Just briefly, if we consider the substantial input we’ve made in the UK with he UTL (and Jim and co. with theirs), and how long it has taken to develop the expertise in SNOMED CT path and lab terminology, Regenstrief and SI probably cannot afford to ignore our developments and work with us in some capacity. I say that because Regenstrief have not got the authoring know-how to develop their content in SNOMED CT, and on the other hand SI do not have authoring expertise in the pathology disciplines. It could be quite a long roadmap for them to set up their operation, so in the meantime I agree with Pete below that we need get our house in order with both UTL and PBCL SNOMED CT modelling because both will be very relevant to the different activities going on internationally: LOINC extension, and E20 conversions. I would be good discuss in more detail when we have a meeting set up.

Thanks
Karim

**Comment by GogsByrn** [comment]
Meeting scheduled 26th Oct
